components/Register.vue

```diff
@@ -21,18 +21,17 @@
         </a>
       </div>
 
-       <span class="RegisterButton">Ouverture de la billeterie le 13 octobre 2022 à 13h00</span>
-<!--      <div>-->
-<!--        <a-->
-<!--          href="https://www.helloasso.com/associations/touraine-tech/evenements/tourainetech-22"-->
-<!--          target="_blank"-->
-<!--          class="RegisterButton button&#45;&#45;link button&#45;&#45;white"-->
-<!--        >Lien vers la billeterie HelloAsso</a>-->
-<!--      </div>-->
+      <div>
+        <a
+          href="https://billetterie.touraine.tech"
+          target="_blank"
+          class="RegisterButton button--link button--white"
+        >Lien vers la billetterie</a>
```

Comment:
We can maybe set a "better" wording like "Achetez votre place!" ?

Reply:
Yes
J'ai repris le wording de l'année dernière